apps/src/p5lab/AnimationPicker/AnimationPickerBody.jsx

```diff
@@ -180,12 +216,14 @@ export default class AnimationPickerBody extends React.Component {
           <div style={animationPickerStyles.navigation}>
             {categoryQuery !== '' && (
               <div style={animationPickerStyles.breadCrumbs}>
-                <span
-                  onClick={this.onClearCategories}
-                  style={animationPickerStyles.allAnimations}
-                >
-                  {'All categories > '}
-                </span>
+                {this.props.navigable && (
+                  <span
+                    onClick={this.onClearCategories}
+                    style={animationPickerStyles.allAnimations}
+                  >
+                    {`${msg.animationPicker_allCategories()} > `}
```

Comment:
we should move the carrot into the `animationPicker_allCategories` string so that it displays correctly in RTL languages (right now, the greater-than sign would be on the incorrect end of the string)

Reply:
Elijah asked me to take it out, do you still want me to put it back in? https://github.com/code-dot-org/code-dot-org/pull/36578#discussion_r490470428

Reply:
The browser is (usually) capable of aligning things like this correctly in RTL, and this way we avoid confusing translators with non-language characters